File: example/javalib/simple/3-custom-class/src/qux/Qux.java
package qux;

public class Qux {
  public static String getLineCount() {
    try {
      return new String(
          Qux.class.getClassLoader().getResourceAsStream("line-count.txt").readAllBytes());
    } catch (java.io.IOException e) {
      return null;
    }
  }

  public static void main(String[] args) throws Exception {
    String lineCount = getLineCount();
    System.out.println("Line Count: " + lineCount);
  }
}

File: example/kotlinlib/simple/3-custom-class/src/qux/Qux.kt
package qux

import java.io.IOException

fun getLineCount(): String? = try {
    ::main.javaClass.classLoader
        .getResourceAsStream("line-count.txt")
        .readAllBytes()
        .toString(Charsets.UTF_8)
} catch (e: IOException) {
    null
}

fun main() {
    var msg = "Line Count: " + getLineCount()
    println(msg)
}
